terraform/modules/bootstrap/bootstrap.tf

```diff
@@ -1,21 +1,26 @@
 module "state_bucket" {
   source = "../s3"
 
-  account_id = data.aws_caller_identity.current.arn
+  account_id = data.aws_caller_identity.current.account_id
   region     = data.aws_region.current.name
   name       = var.s3_bucket_name
 
   tiering_noncurrent_days = 10
 
   sns_arn = module.sns.topic_arn
 
+  cross_account_arns = [
+    "arn:aws:iam::255773200490:role/prod-github-oidc-deploy",
+    "arn:aws:iam::255773200490:role/github-oidc-pull-request"
+  ]
+
```

Comment:
what are these for?

Reply:
To allow prod to access the state of non prod